### ISSUE TITLE
Steam Beta Client Major Update

### ISSUE THREAD

**Issue by FlamingFox911** [DirtDiglett/Pressure-for-Steam#484]
What is says on the tin. With the recent major [update](https://steamcommunity.com/groups/SteamClientBeta#announcements/detail/1656643072593941317) to chat client, this broke some key UI features.

I no longer have the time to do further updates with this project.

EDIT: Just did some quick [research](https://www.reddit.com/r/Steam/comments/8qlk0h/major_steam_client_beta_update_for_61218_the/e0kw15j/), if these updates continue, its entirely possible we will see the cessation of skinning.

**Comment by Js41637** [comment]
I will be updating pressure for steam to get some basic functionality back, I already have it mostly usable

![image](https://user-images.githubusercontent.com/6258213/41338200-cdd3acc4-6f34-11e8-874b-7e1e75d07621.png)


But yeah the new chat/friends stuff can't be styled

**Comment by pt300** [comment]
When can we expect changes to be pushed?

**Comment by pt300** [comment]
just noticed second branch

**Comment by FlamingFox911** [comment]
Thanks, Js. Much appreciated. Finding it hard to find time nowadays.

**Comment by FURTEK-SV** [comment]
It can be styled and rewritten but now it'll have to be through CSS and JS so it's getting pretty complicated in that regard.

**Comment by Js41637** [comment]
Yeah I know how it works but I highly doubt it can be edited, the files are stored in the Steam Client but I think when you start the client the bootstrap will overwrite them as they can be updated independently of the main client over the network. Basically, they cannot be edited.

**Comment by FURTEK-SV** [comment]
Is it possible to use an injection method to rewrite on the fly or is injecting into the clients bootstrap too risky and at the possibility to cause major issues with the functionality?
